src/locales/zh.json

```
@@ -67,7 +67,7 @@
       "block_mined": "该地址已铸造的区块数量",
       "args": "在 CKB 中，公钥信息可以存储在脚本结构的 args 字段中。args 在脚本执行时使用，例如，当脚本用于验证私钥的所有者时，args 可以是公钥哈希。",
       "capacity_change": "该地址在交易前后 CKB 余额的变化值",
-      "estimated_apc": "预估年收益率 XXXX",
+      "estimated_apc": "预估年收益率",
```

```suggestion
      "estimated_apc": "预估年补贴率",
```

`预估年补贴率` maps to `APY(Annual percentage Yield)`, `APC` is slightly different from `APY` because `APY` has a strong economic attribute.